app/services/ReleasedServices.scala

```diff
@@ -1,7 +1,8 @@
 package services
 
-import javax.inject.Inject
-import models.{Movie, MovieWithID}
+import javax.inject.{Inject, Singleton}
+import models.MovieWithID
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277123)